src/main/scala/com/databricks/labs/overwatch/pipeline/Bronze.scala

```diff
@@ -95,6 +95,37 @@ class Bronze(_workspace: Workspace, _database: Database, _config: Config)
     Helpers.parClone(cloneSpecs)
 
   }
+  def snapshotStream(
```

Comment:
Add code docs on what is expected to be passed in for these variables and an English explanation of what this function does.